syntax = "proto3";

message ImageFileReq {
  double time_sent = 1;
  bytes  data = 2;
}

message ImageFileRecv {
  double time_recv = 1;
}

message PingServer {
}

message PingClient {
  double time_sent = 1;
}

message SendImage {
  bytes data = 1;
}

message SendImageAck {
  double time = 1;
}